src/plus/integrations/integrationService.ts

```diff
@@ -127,7 +127,7 @@ export class IntegrationService implements Disposable {
 	}
 
 	private onUserCheckedIn() {
-		void this.syncCloudIntegrations(false);
+		void this.syncCloudIntegrations(true);
```

Comment:
@sergeibbb This will cause check-ins to always force reconnection to providers, even when the user manually disconnects and disables them.

check-in is considered a passive sync case and should never force reconnections, because it happens automatically (in my repro, I can trigger it by refreshing the account view).

![checkin-forces-reconnection](https://github.com/user-attachments/assets/ce67a450-9389-4357-bc4f-2526bb12dcb8)

Instead, we should check the logout flow and ensure that integrations can be disconnected "softly" when a logout or some other passive sync case triggers the disconnect. Then those integrations would reconnect upon login even when `forceConnect` is `false` on `syncCloudIntegrations`.

Reply:
fixed